digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/RoadLinkServiceSpec.scala

```diff
@@ -864,4 +864,26 @@ class RoadLinkServiceSpec extends FunSuite with Matchers with BeforeAndAfter {
     }
   }
 
+  test("verify the output of change info") {
+    val oldLinkId = 1l
+    val newLinkId = 2l
+    val changeInfo = ChangeInfo(Some(oldLinkId), Some(newLinkId), 123l, 5, Some(0), Some(1), Some(0), Some(1), 144000000)
+    val boundingBox = BoundingRectangle(Point(123, 345), Point(567, 678))
+    val mockVVHClient = MockitoSugar.mock[VVHClient]
+    val mockVVHRoadLinkClient = MockitoSugar.mock[VVHRoadLinkClient]
+    val mockVVHChangeInfoClient = MockitoSugar.mock[VVHChangeInfoClient]
+    val service = new TestService(mockVVHClient)
+    OracleDatabase.withDynTransaction {
```

Comment:
Maybe but if you look at all the tests in the test suit all of them are not using the overided version.